activejob/test/support/integration/adapters/sneakers.rb

```diff
@@ -84,7 +84,7 @@ def bunny_publisher
     end
 
     def bunny_queue
-      @queue ||= bunny_publisher.exchange.channel.queue "integration_tests", durable: true
+      @queue ||= bunny_publisher.send(:exchange).channel.queue "integration_tests", durable: true
```

Comment:
Done. Exchange moved back, and released new v1.0.4 to reflect this (and other previous improvements).
Feel free to LMK if there's anything more I can do to help, thanks!